packages/helm/symphony/files/oss/delete-objects.sh

```diff
@@ -1,29 +1,117 @@
 #!/bin/env bash
 set -e
-TIMEOUT="30m"
+TIMEOUTINSTANCE="15m"
```

Comment:
Better to rename for TIMEOUTDEPLOYMENT="15m", or use two separate variables for two deletions.